.vscode/tasks.json

```diff
@@ -9,7 +9,7 @@
         {
             "label": "v5_hal Firmware Build",
             "type": "shell",
-            "command": "catkin_make v5_hal_firmware_build",

```

Comment:
If possible I would keep the WSL firmware build as a command, but add it as "v5_hal WSL Firmware Build" for those of us still on the old system

Reply:
fixed in latest commit